hw1103compaundviewnavigationdrawer/src/full/java/com/tashad16a/gmail/full/fragment/CompoundPageFragment.kt

```diff
@@ -0,0 +1,39 @@
+package com.tashad16a.gmail.full.fragment
+
+import android.content.Context
+import android.support.annotation.Nullable
+import android.util.AttributeSet
+import android.widget.LinearLayout
+import android.widget.TextView
+import com.tashad16a.gmail.hw1103compaundviewnavigationdrawer.R
+
+class CompoundPageFragment : LinearLayout {
```

Comment:
if code of classes the same for full and demo app, better leave it in src/main/java folder